### ISSUE TITLE
Resume fails after network errors

### ISSUE THREAD

**Issue by AVR2CO5** [downthemall/downthemall#94]
A 17Gb download from put.io failed partway through with a "network error". When I selected "resume", it started again from the beginning, but the .part file for the failed download still exists on my hard drive.

Resume worked correctly from put.io in the previous incarnation of DownThemAll.

OSX 10.11.6, Firefox 69.0

**Comment by nmaier** [comment]
Yes, that's the Firefox internal downloader we have to use throwing errors. Regular "Save As" uses it too but you might get lucky still. Hope your download finishes.

**Comment by BioSehnsucht** [comment]
> 
> 
> Upstream: https://bugzilla.mozilla.org/show_bug.cgi?id=1580920

Apparently this got triaged as P5 (Won't fix but accepting patches for it) upstream ? Fixing it is beyond me, but thought I'd note that here for anyone who is hoping Firefox fixes it... :(

**Comment by SteenG** [comment]
At least Firefox can pickup from where it lost the download while DownThemAll begins all over (again and again).

**Comment by ZaDarkSide** [comment]
2021 still not solved on Chrome 😎

**Comment by lee321987** [comment]
2022 still a problem.

**Comment by spiralofhope** [comment]
2023 still-- actually I don't know but I wanted to participate.

**Comment by MiMoHo** [comment]
Can confirm the problem exists in v4.8.1 with Firefox for Mac v110 in contrary to the Firefox in-built downloader that is able to resume failed individual token downloads after network issues.
Issue got duplicated by #419 with a more comprehensive description. Please merge into here @nmaier

**Comment by kiwichick888** [comment]
Still happening in v4.11.1 with Firefox 116.0.3.

**Comment by HieronymusCH** [comment]
And still happening.
123.0 (64-Bit)
Mozilla Firefox for Arch Linux
archlinux - 1.0

**Comment by kiwichick888** [comment]
> And still happening. Now the *.part file even disappears after failing.
> 
> 123.0 (64-Bit) Mozilla Firefox for Arch Linux archlinux - 1.0

I honestly don't think communications of any kind are being read. I've been trying for the last couple of years via email and forums but not a single reply or even acknowledgement of receipt. It galls me that the developer is still asking for donations to continue development but won't communicate with users.
